force-app/main/default/lwc/gseuTrailhead/gseuTrailhead.html

```diff
@@ -2,7 +2,7 @@
     <lightning-card>
         <lightning-layout horizontal-align="space" multiple-rows="true">
             <lightning-layout-item size="12" class="slds-text-align_center">
-                <img class="" src={imgUrl} alt=""/>
+                <img class="" src={imgUrl} alt="Trailhead Logo"/>
```

Comment:
Need a custom label here. Please create one for "Trailhead Logo" if it doesn't already exist. Thanks!

Reply:
Agreed. Change done. Thank you, Prince